packages/chats/src/BubblePlainText.tsx

```diff
@@ -26,6 +28,29 @@ import { MessageStatus, OnLongPressText, OnPressUrl } from './types';
 import type { ChatPlainTextMessage, PlainTextMessage } from './types';
 import { useBubblePosition, useBubbleContainerStyle } from './useBubblePosition';
 import { useBubbleBackgroundColor, useBubbleRoundedCornerStyle } from './useBubbleStyle';
+import { URI_TON_SURF } from './constants';
+
+const renderParsedText = (matchingString: string, matches: string[]) => {
+    if (matches[1]) {
+        const domain = matches[0].replace(matches[1], '').replace(matches[2], '');
+        if (domain === URI_TON_SURF) {
+            const clearUrl = matches[0].replace(matches[1], '');
+            return matchingString.replace(matches[0], '') + decodeURI(clearUrl);
+        }
```

Comment:
To be honest, it's not clear to me what's going on here. As far as I see we again tend to display the URI at the end of the message, am I right?